src/RoleObjects/MultiLineTextBoxData.js

```diff
@@ -329,4 +348,35 @@ export default class MultiLineTextBoxData extends AccessibilityObject {
     event.selectionDirection = evt.currentTarget.selectionDirection;
     this._displayObject.dispatchEvent(event);
   }
+
+  /**
+   * event handler for when the element gets input
+   * @param {Event} evt
+   */
+  _onInput(evt) {
+    this._onChange(evt);
+    this._onSelect(evt);
+  }
+
+  /**
+   * event handler for when keyboard key is lifted Up
+   * @param {Event} evt
+   */
+  _onKeyUp(evt) {
+    if (this.enableKeyEvents) {
+      super._onKeyUp(evt);
+      if (evt.defaultPrevented) {
+        return;
+      }
+    }
+
+    if (
```

Comment:
I haven't pulled the branch to test myself if this is needed, but is there a need to call out other keystroke that can affect the selection such as home and end, along with whether shift is being held down so that selectionStart and selectionEnd are different values?

Reply:
@mnewcomb314 We can add Home and End keystrokes as well but, for when shift is being held down `onSelect` event listener is directly being called so we don't have to specifically check for that case!

Reply:
Thanks!  And excellent news about the shift case!